typing/includemod.ml

```diff
@@ -679,11 +679,13 @@ module Illegal_permutation = struct
   (* Find module type at position [path] and convert the [coerce_pos] path to
      a [pos] path *)
   let rec find env ctx path mt = match mt, path with
-    | (Mty_ident p | Mty_alias p), _ ->
+    | Mty_ident p, _ ->
         begin match (Env.find_modtype p env).mtd_type with
         | None -> raise Not_found
         | Some mt -> find env ctx path mt
         end
+    | Mty_alias p, _ ->
+        find env ctx path (Env.find_module p env).md_type
```

Comment:
It seems better to use an assert false here: this case cannot happen because for module aliases, equivalence up to permutation is the same as equivalence up to permutations of nonruntime components.

Reply:
Good point! I've pushed an update with this change.